mempool/policy.go

```diff
@@ -253,6 +246,20 @@ func IsDust(txOut *wire.TxOut, minRelayTxFee btcutil.Amount) bool {
 		totalSize += 107
 	}
 
+	return 3 * int64(totalSize)
```

Comment:
Why is the factor of `3` here retained? When in the original dust calculation, this value is used to ensure that the amount spent on fees to spend an output isn't _greater_ than 1/3 of the relay fee.

Reply:
Ah ok, we chatted offline and derived why this is in place.